style.css

```diff
@@ -0,0 +1,225 @@
+/*==========================================================
+    FONTS
+==========================================================*/
+@import url('https://fonts.googleapis.com/css?family=Roboto+Condensed:300,400,700&display=swap');
+/*==========================================================
+    VARIABLES
+==========================================================*/
+:root {
+    --news-red: #f72210;
+    --ethic-white: #fff;
+}
+/*==========================================================
+    GENERAL TAGS
+==========================================================*/
+body {
+    font-family: Arial,Helvetica,sans-serif;
+}
+
+.hid {
+    display: none;
```

Comment:
Consider using a `Bootstrap` class name for this selector element and in all other occurrences instead.

Bootstrap class names can be used instead. See [Display](https://getbootstrap.com/docs/4.1/utilities/display/) in bootstrap documentation.

> Remember to use Bootstrap CSS classes wherever it is possible - this is the main point of this project.

#### Please, lets do this for all other occurrences. :blush:

Reply:
Done!